prod-eu-west/services/metrics-api/main.tf

```diff
@@ -35,7 +35,12 @@ resource "aws_lb_listener_rule" "metrics-api" {
 
   condition {
```

Comment:
Make sure the priority is not already taken and is before any catch-all rule for api.datacite.org.